### PR TITLE
:lipstick: News section #7

### PR DESCRIPTION
**Pull request by Sharoun-Madoya**
# Description
Create Home Page News Section Component #7

Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [#7 ](https://github.com/italanta/elewa-website-html/issues/7).

Fixes # (issue) - *Example*: Fixes #10, #20 NOTE: GitHub issue numbers should be prefixed with **#** to aid with issue tracking.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/css/style.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  
+body {
+    font-family: Arial, Helvetica, sans-serif;
+    background-color: #f2f2f2;
+    padding: 20px;
+  }
+
+h1{
+ margin-top: 20vh;
+ margin-left: 80px;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: use shorthand

File: src/css/style.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+ font-size: 4em;
+ margin-bottom: 20px;
+}
+
+h4{
+     margin-top: 50px;
+    font-size: 25px;
+    margin-left: 30px;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: here also, for the margin

File: src/css/style.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    /* text-align: center; */
+}
+
+.text{
+    margin-left: 80px;
+    color: grey;
+}
+.read-more {
+    margin-top: 50px;
+    margin-left: 10%;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: here

File: src/css/style.css [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    display: block;
+    
+}
+.read-more a{
+    color: grey;
+
+}
+.read-more3 {
+    margin-top: 50px;
+    margin-left: 15%;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: here

File: src/css/style.css [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    font-size: 20px;
+    display: block;   
+}
+.read-more3 a{
+    color: grey;
+}
+
+.spacer{
+    margin-top: 50px;
+    margin-left: 50px;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: here

**Review by JAPHETHNYARANGA — CHANGES_REQUESTED** [review]
Good attempt, edit pr name to have the issue number, change emji to a lipstick. And push only the required files. Don't forget to attach screenshots. Thank you.

File: src/css/style.css (hunk 5 above)
Comment (Sharoun-Madoya) [reply src/css/style.css]: Done the Changes

**Comment by Sharoun-Madoya** [comment]
> # Description
> Create Home Page News Section Component #7
> 
> Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [#7 ](https://github.com/italanta/elewa-website-html/issues/7).
> 
> Fixes # (issue) - _Example_: Fixes #10, #20 NOTE: GitHub issue numbers should be prefixed with **#** to aid with issue tracking.
> Screen Shot




> ## Type of change
> Please delete options that are not relevant.
> 
> * [x]  New feature (non-breaking change which adds functionality)
> * [x]  This change requires a documentation update
> 
> ## Checklist
> * [x]  My code follows the style guidelines of this project
> * [x]  I have performed a self-review of my code
> * [x]  I have made corresponding changes to the documentation
> * [x]  My changes generate no new warnings

**Comment by JAPHETHNYARANGA** [comment]
This is good work, the design is nice.Good work. Just closing the pr since the issue is closed

**Pull request closed without merging**